dynawo/sources/Test/initXml.cpp

```diff
@@ -1,14 +1,14 @@
 //
-// Copyright (c) 2021, RTE (http://www.rte-france.com)
+// Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
```

Comment:
When a new file is created the year of the copyright should be the year it is created so don't add 2015, and this file was created in 2021.